gordon/plugins_loader.py

```diff
@@ -64,13 +64,13 @@
 PLUGIN_NAMESPACE = 'gordon.plugins'
 
 
-def _init_plugins(active_plugins, installed_plugins, plugin_configs):
+def _init_plugins(active, installed, configs, kwargs):
     inited_plugins, inited_plugin_names, errors = [], [], []
-    for plugin_name, unloaded_plugin in installed_plugins.items():
-        if plugin_name not in active_plugins:
+    for plugin_name, unloaded_plugin in installed.items():
+        if plugin_name not in active:
             continue
         plugin_class = unloaded_plugin.load()
```

Comment:
Not in the diff, but what does load() do and where is it defined?  It's not in this repo.

Reply:
(Maybe add a comment)

Reply:
The `unloaded_plugin` is actually an instance of pkg_resources's [`EntryPoint`](http://setuptools.readthedocs.io/en/latest/pkg_resources.html#entrypoint-objects)s.